packages/react/src/components/Cta/CTA.js

```diff
@@ -0,0 +1,115 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { ArrowDown20, ArrowRight20, Launch20 } from '@carbon/icons-react';
+import { ButtonGroup } from '../../patterns/sub-patterns/ButtonGroup';
+import { CardLink } from '../../patterns/sub-patterns/CardLink';
+import { FeaturedLink } from '../../patterns/blocks/FeaturedLink';
+import { LinkWithIcon } from '../LinkWithIcon';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { smoothScroll } from '@carbon/ibmdotcom-utilities';
+/**
+ * CTA component
+ *
+ * @param {object} props props object
+ * @param {string} props.style style ( text | card | button | feature ).
+ * @param {object} props.type tyle ( jump | local | external ).
+ * @param {object} props.cta cta object which includes href,text, copy, card and image, button  properties.
+ * @returns {*} CTA Component
+ */
+const CTA = ({ style, type, ...cta }) => {
+  switch (style) {
+    case 'card':
+      return (
+        <CardLink
+          {...cta}
+          icon={iconSelector(type)}
+          target={external(type)}
+          onClick={e => jump(e, type)}
+        />
```

Comment:
For this one we need to lock it down until we can figure out card decorators.

I believe all we need is the heading, and we should probably give it a `className`.

```jsx
<CardLink
  copy={cta.copy}
  className={`${prefix}--card-cta`}
  icon={iconSelector(type)}
  target={external(type)}
  onClick={e => jump(e, type)}
/>
```

Then we might want to add some `SCSS` (it could even be apart of the card sub-pattern possibly for now where we define `heading-02` for the copy.

I think you can use copy or heading here, but I'm recommending `copy` since this is essentially just like a large text link.

Reply:
here I have a doubt will sync up with you on slack.